.github/workflows/autoformat.yml

```diff
@@ -113,7 +113,11 @@ jobs:
           pip install -e .
       - if: steps.diff.outputs.protos == 'true'
         run: |
-          python ./dev/generate_protos.py
+          # Run the script multiple times. The changes generated by the first run
+          # may trigger additional changes, which need to be applied in subsequent runs.
+          for i in {1..3}; do
```

Comment:
Is 3 the max? Or just came up with a number that seems about right? 😅

Reply:
2 should be enough. 3 for safety.